authpass/lib/ui/screens/password_list.dart

```diff
@@ -363,6 +364,10 @@ class _PasswordListContentState extends State<PasswordListContent>
 
 //  final _isolateRunner = IsolateRunner.spawn();
 
+  late AutofillService _autofill;
```

Comment:
`_autofill` is not used outside of `_updateAutofillPrefs()` so we could probably get rid of that attribute?

Reply:
fixed.